www/app/app.component.ts

```diff
@@ -0,0 +1,74 @@
+import { Component, OnInit } from '@angular/core';
+import { Http, Response } from '@angular/http';
+import { Observable } from 'rxjs/Rx';
+import { HTTP_PROVIDERS } from '@angular/http';
+
+@Component({
+  selector: 'my-app',
+  templateUrl: 'app/app.component.html',
+  styleUrls: ['app/app.component.css'],
+  providers: [ HTTP_PROVIDERS ]
+})
+
+export class AppComponent implements OnInit {
+  message: string = null;
+  currentStep: number = 0;
+  response: any;
+  previousCommand: string = '';
+  instructions: any;
+
+  constructor(private http: Http) { }
+
+  ngOnInit() {
+    this.getInstructions(this.currentStep);
+  }
+
+  evalInput(value) {
+    if (value.match(/^inspec exec\s*examples\/profile\s*/)) {
```

Comment:
During generation of the commands, we probably want to have json file that tracks the commands. We could use https://github.com/chef/inspec/blob/master/tasks/command_simulator.rb#L4 as an external json. This enables us to use it as a reference in this app and as a list of commands for the rake task